test/relayclient/KnownRelaysManager.test.ts

```diff
@@ -390,4 +396,30 @@ contract('KnownRelaysManager 2', function (accounts) {
       assert.equal(sortedRelays[1][2].pctRelayFee, '4')
     })
   })
+
+  describe('#getAuditors()', function () {
+    const auditorsCount = 2
+    let knownRelaysManager: KnownRelaysManager
+    before(function () {
+      const activeRelays: RelayInfoUrl[] = [{ relayUrl: 'alice' }, { relayUrl: 'bob' }, { relayUrl: 'charlie' }]
+      knownRelaysManager = new KnownRelaysManager(contractInteractor, logger, configureGSN({ auditorsCount }))
+      sinon.stub(knownRelaysManager, 'allRelayers').value(activeRelays)
+    })
+
+    it('should give correct number of unique random relay URLs', function () {
+      const auditors = knownRelaysManager.getAuditors()
+      const unique = auditors.filter((value, index, self) => {
```

Comment:
nitpic: took me a little to understand. I'd simply do `unique = new Set(auditors)`

Reply:
Not part of this PR